app/controllers/pieces_controller.rb

```diff
@@ -13,10 +13,13 @@ def update
     #will need to do a proper game end
 
     king_opp = @game.pieces.where(:type =>"King").where.not(:user_id => @game.turn_user_id)[0]
+    king_current = @game.pieces.where(:type =>"King").where(:user_id => @game.turn_user_id)[0]
     game_end = false
     if king_opp.check?(king_opp.x_coord, king_opp.y_coord).present?
+      render json: {status: "continue", code: 100, message: "You are in check!"}
```

Comment:
@denisekelsey Currently, with the render line 19, two tests are failing due to render/redirect being called multiple times. When i remove that line, the tests pass. 

Would it be okay if i removed line 19?  The flash notice code in line 25 will be able to alert the user if the king is in check.

Reply:
absolutely.  I'm pretty sure that's not my code anyway.  I thought it was yours.  I just added  the exclamation point :)

Reply:
I'll remove it and merge